disemvoweling/vowels.css

```diff
@@ -0,0 +1,50 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans, Lato);
+
+body {
+    font-family: 'Lato', sans-serif;
+    font-size: 26px;
+    color: #fafafa;
+    background-image: url("letters.jpg");
+    no-repeat center center fixed;
+    -webkit-background-size: cover;
```

Comment:
I forget, is there a way that we can condense all of the background sizes? I'm not sure.